test/envtest/konnect_entities_kongcertificate_test.go

```diff
@@ -180,63 +180,6 @@ func TestKongCertificate(t *testing.T) {
 		eventuallyAssertSDKExpectations(t, sdk.CertificatesSDK, waitTime, tickTime)
 	})
 
-	t.Run("should handle konnectID control plane reference", func(t *testing.T) {
```

Comment:
Given that we will want to eventually support these, how about adding an issue for `konnectID` support and linking it in `t.Skip()` call?

This way we'll save ourselves some work when we'll get to introducing `konnectID` support. WDYT?

Reply:
Done with adding the `t.Skip`.